Makefile

```
@@ -105,8 +105,10 @@ cmakelint:
 		--step 'Run cmakelint'
 
 clang_tidy:
```

could this be clang-tidy to be consistent with the previous GHA workflow name? 
```suggestion
clang-tidy:
```